src/Castle.Windsor/Castle.Windsor-VS2017.csproj

```diff
@@ -1,41 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
-  </PropertyGroup>
-  
-  <Import Project="..\..\common.props"></Import>
```

Comment:
Please keep this order in all csproj files, first target frameworks then import of common.props.